### PR TITLE
resetting password in admin view

### PR DESCRIPTION
**Pull request by pawc**

### CODE REVIEW

**Comment by pawc** [comment]
ready to merge if you like it

File: client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java [hunk 1]
```diff
@@ -126,7 +134,7 @@ private void editUser(){
             EditUserPane= (AnchorPane) fxmlLoader.load();
         }
         catch(IOException e){
-            logger.debug("Error loading frame to edit user " + e.toString());
+            logger.error("Error loading frame to edit user " + e.toString());
```
Comment (paddatrapper) [inline client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java]: Instead of logging the e.toString(), rather just pass the error as a parameter so the entire trace stack is logged

File: client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java [hunk 2]
```diff
@@ -252,6 +260,49 @@ public void updateTable(){
...
+            alert = new Alert(AlertType.ERROR);
+            alert.setTitle("Password reset");
+            alert.setHeaderText("Error during password reset");
+            alert.setContentText("Error occured on the server side");
+            alert.showAndWait();
+            break;
+          }  
+    }
+    
+    private String randomPassword() {
```
Comment (paddatrapper) [inline client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java]: This should rather be in the Administrator class. I try to keep everything that is not directly related to UI code in the other classes

File: client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java [hunk 3]
```diff
@@ -123,5 +123,19 @@ private InputToModelParseResult validate(Map<String, Object> inputMap){
 
        return result;
     }
+
+    public int resetPassword(Map<String, Object> inputMap) {
```
Comment (paddatrapper) [inline client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java]: Out of curiosity why did you choose a map instead of 2 parameters?

Comment (pawc) [inline client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java]: to keep the convention i guess

Comment (paddatrapper) [inline client/src/main/java/com/kritsit/casetracker/client/domain/services/Administrator.java]: I think it would be simpler to take it as 2 parameters. The reason for the Map was because the data was yet to be validated to fit the model
